packages/suite/src/views/settings/coins/index.tsx

```diff
@@ -69,6 +69,36 @@ const CoinSymbol = styled.div`
     padding-top: 2px;
 `;
 
+const Coin = ({ network }: { network: Network | ExternalNetwork }) => (
+    <CoinWrapper>
+        <CoinLogo size={24} symbol={network.symbol} />
+        <CoinName> {network.name}</CoinName>
+        <CoinSymbol> {network.symbol.toUpperCase()}</CoinSymbol>
+    </CoinWrapper>
+);
+
+const AdvancedSettings = styled.div`
+    cursor: pointer;
+    font-size: ${variables.FONT_SIZE.TINY};
+    /* todo: not in variables but is in design */
+    font-weight: 500;
+    margin-right: 4%;
+    min-width: 120px;
+    visibility: hidden;
+`;
+
+const SettingsIcon = styled(Icon)`
+    position: relative;
+    top: 2px;
+    right: 4px;
```

Comment:
just a tip: you could  wrap the text "Advanced Settings" to its own div and to the parent container of the text and the icon add `display:flex; align-items: center`. it would be more future-proof since it will keep items centered even with different font sizes.